Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.6-onbuild
+MAINTAINER Praekelt Foundation <dev@praekeltfoundation.org>
+
+EXPOSE 3000
+VOLUME /logs
+
+CMD ["go-wrapper", "run", \
```

Comment:
Logdriver executable also available just as `app` because it's copied in to `/go/src/app`... Weird golang path stuff :/